src/data/list/min_max.lean

```diff
@@ -271,4 +271,61 @@ theorem minimum_eq_coe_iff {m : α} {l : list α} :
   minimum l = m ↔ m ∈ l ∧ (∀ a ∈ l, m ≤ a) :=
 @maximum_eq_coe_iff (order_dual α) _ _ _
 
+section fold
```

Comment:
```suggestion
section fold
/-! Note: since there is no typeclass for both `linear_order` and `has_top`, nor a typeclass dual
to `canonically_linear_ordered_add_monoid α` we cannot express these lemmas generally for
`minimum`; instead we are limited to doing so on `order_dual α`. -/
```